No idea why the build got stuck in javadoc creation:

```
15:59:39.080 [DEBUG] [org.gradle.process.internal.ExecHandleRunner] waiting until streams are handled...
15:59:39.080 [INFO] [org.gradle.process.internal.DefaultExecHandle] Successfully started process 'command '/opt/java/sdk/Sun/x86_64/jdk1.6.0_45/bin/javadoc''
Build timed out (after 35 minutes). Marking the build as aborted.
Build was aborted
Recording test results
ERROR: Publisher hudson.tasks.junit.JUnitResultArchiver aborted due to exception
hudson.util.IOException2: remote file operation failed: /opt/jdk/jdk1.6.latest at hudson.remoting.Channel@430f5df2:655c59df
    at hudson.FilePath.act(FilePath.java:907)
    at hudson.FilePath.act(FilePath.java:884)
    at hudson.FilePath.exists(FilePath.java:1316)
    at com.cloudbees.hudson.plugins.defaults.tools.PreInstalledToolLocator.getToolHome(PreInstalledToolLocator.java:48)
    at hudson.tools.ToolLocationNodeProperty.getToolHome(ToolLocationNodeProperty.java:107)
    at hudson.tools.ToolInstallation.translateFor(ToolInstallation.java:204)
    at hudson.model.JDK.forNode(JDK.java:126)
    at hudson.model.AbstractProject.getEnvironment(AbstractProject.java:359)
    at hudson.model.Run.getEnvironment(Run.java:2173)
    at hudson.model.AbstractBuild.getEnvironment(AbstractBuild.java:924)
    at hudson.tasks.junit.JUnitResultArchiver.perform(JUnitResultArchiver.java:130)
    at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
    at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:795)
    at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:767)
    at hudson.model.Build$BuildExecution.post2(Build.java:183)
    at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:716)
    at hudson.model.Run.execute(Run.java:1690)
    at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:46)
    at hudson.model.ResourceController.execute(ResourceController.java:88)
    at hudson.model.Executor.run(Executor.java:246)
Caused by: hudson.remoting.ChannelClosedException: channel is already closed
    at hudson.remoting.Channel.send(Channel.java:541)
    at hudson.remoting.Request.call(Request.java:129)
    at hudson.remoting.Channel.call(Channel.java:739)
    at hudson.FilePath.act(FilePath.java:900)
    ... 19 more
Caused by: java.io.IOException
    at hudson.remoting.Channel.close(Channel.java:1027)
    at hudson.slaves.ChannelPinger$1.onDead(ChannelPinger.java:110)
    at hudson.remoting.PingThread.ping(PingThread.java:120)
    at hudson.remoting.PingThread.run(PingThread.java:81)
Caused by: java.util.concurrent.TimeoutException: Ping started on 1404255757101 hasn't completed at 1404255997112
    ... 2 more
Putting comment on the pull request
Finished: ABORTED
```

--- a/rx-netty/src/main/java/io/reactivex/netty/client/PoolStatsImpl.java
+++ b/rx-netty/src/main/java/io/reactivex/netty/client/PoolStatsImpl.java
@@ -30,7 +30,7 @@ import java.util.concurrent.atomic.AtomicLong;
 @Deprecated
 public class PoolStatsImpl implements PoolStats, CompositeHttpClientBuilder.CloneablePoolStatsProvider {
 
-    private final AtomicLong idleConnections;
+    private final AtomicLong idleConnections; // LongAdder backport is not distributed with netty anymore. So moving to AtomicLong temporarily before we remove this class.
     private final AtomicLong inUseConnections;
     private final AtomicLong totalConnections;
     private final AtomicLong pendingAcquires;

